docs/_gks/about/changelog-v2.21/index.en.md

```diff
@@ -0,0 +1,46 @@
+---
+title: GKS Changelog v2.21
+lang: en
+permalink: /gks/about/changelog-v2.21/
+nav_order: 600
+parent: About GKS
+---
+
+## Supported Kubernetes Versions
+
+With the current release, we support the following Kubernetes versions:
+
+* 1.22.15
+* 1.23.12
+* 1.24.6
+
+## End of Life Announcements
+
+We will end the support of Kubernetes version 1.22 on October 10, 2022.
+
+Make sure to upgrade all existing clusters running on version 1.22 at least to version 1.23 before that date.
+
+## New Features
+
+### Project overview site
+
+We added a project overview site which shows a condenst overview the project.
```

Comment:
We added a project overview page that gives an overview of the content of the project.